packages/core/src/pagination/pagination-button.element.scss

```diff
@@ -0,0 +1,51 @@
+@import './../styles/tokens/generated/index';
+
+:host {
+  --height: #{$cds-global-space-9};
+  --min-width: #{$cds-global-space-9};
+  --padding: 0;
+  --border-color: var(--color);
+  --border-width: 0;
+  --border-radius: #{$cds-alias-object-border-radius-100};
+  --box-shadow-color: #{$cds-alias-object-opacity-100};
```

Comment:
We should probably make this just `--box-shadow` and put the full style from line 27 in it. that would give users more flexibility to customize it or remove it.